content/tutorials/pubnub-adaptor.textile

```diff
@@ -0,0 +1,199 @@
+---
+title: Pubnub Protocol Adaptor Tutorial
+section: tutorials
+index: 15
+---
+
+The Pubnub protocol adaptor allows you to interact with Ably using the Pubnub protocol.  You can use this to gradually migrate across from Pubnub to Ably. Rather than having to rewrite all your apps to use Ably client libraries at once, you can use the protocol adaptor, then migrate across to Ably client libraries one by one at your leisure, to take advantage of "Ably features not supported by the Pubnub client libraries":https://www.ably.io/features, such as "the ability to use websockets":https://support.ably.io/solution/articles/3000044831-which-transports-are-supported%2D (Pubnub client libraries support long-polling only). Alternatively, you can use a Pubnub client library on some platform for which no native Ably client library is available yet, such as Arduinos.
```

Comment:
"Ably features not supported by the Pubnub client libraries" should link to https://www.ably.io/compare/ably-vs-pubnub